nixos/modules/services/web-apps/keycloak.nix

```diff
@@ -340,16 +339,13 @@ in
               '';
             };
 
-            hostname-strict-backchannel = mkOption {
```

Comment:
There's another mention of `hostname-strict-backchannel` in `keycloak.md` that'll need updating.

Reply:
Done